deploy/charts/kaudit/templates/kaudit.yaml

```diff
@@ -99,11 +99,18 @@ spec:
       # Pod Security
       serviceAccountName: alcide-k8s-kaudit-{{ .Values.clusterName }}      
       securityContext:
+{{- if eq .Values.k8s.openshift "true" }}
+        # for openshift use be in the ranges: [1000590000, 1000599999]
+        runAsNonRoot: true
+        runAsUser: 1000590000
```

Comment:
@Amirilw no need to split for open shift - use 1000590000 instead 10001

Reply:
this is running with native k8s, a new PR Will be issued